google/cloud/opentelemetry/internal/recordable_test.cc

```diff
@@ -442,13 +451,19 @@ TEST(Recordable, TruncatesEventName) {
 TEST(Recordable, DropsNewEventAtLimit) {
   auto now = std::chrono::system_clock::now();
 
-  auto rec = Recordable(Project(kProjectId));
+  auto rec = Recordable(Project(kProjectId), TestGenerator());
   for (std::size_t i = 0; i != kSpanAnnotationLimit + 10; ++i) {
     rec.AddEvent("event" + std::to_string(i), now, KVIterable({}));
   }
   auto proto = std::move(rec).as_proto();
   EXPECT_EQ(proto.time_events().dropped_annotations_count(), 10);
   EXPECT_THAT(proto.time_events().time_event(), SizeIs(kSpanAnnotationLimit));
```

Comment:
s/EXPECT/ASSERT/

Reply:
Although I think I would rather you say:

```
AllOf(
  SizeIs(n),
  Contains(TimeEvent(_, Annotation("event0"))),
  Contains(TimeEvent(...)))
```

than subsequently access the elements